Doc/library/unittest.mock.rst

```diff
@@ -860,6 +860,18 @@ object::
     3
     >>> p.assert_called_once_with()
 
+.. caution::
+
+    If an :exc:`AttributeError` is to be raised by :class:`PropertyMock`, it will be interpreted as a missing descriptor and following :meth:`__getattr__` call will be called on the parent mock. ::
```

Comment:
- Please keep the line length under 80 characters
- The full name of the method is `object.__getattr__`; use a `~` to link to it but display only the last part.
- The `::` will add a colon; adding `.` before it is not necessary.
```suggestion
    If an :exc:`AttributeError` is raised by :class:`PropertyMock`,
    it will be interpreted as a missing descriptor and
    :meth:`~object.__getattr__` will be called on the parent mock::
```